cmd/manager/main.go

```diff
@@ -197,6 +197,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = (&replicationController.VolumeGroupReplicationReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
```

Comment:
@Madhu-1, I think we forgot to add the `replicationv1alpha1.VolumeGroupReplication` CRD scheme?